pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -13,7 +11,7 @@ let
       inherit sha256;
     };
 
-    ARCH = stdenvNoCC.hostPlatform.platform.kernelArch;
+    ARCH = stdenvNoCC.hostPlatform.platform.kernelArch or (throw "missing kernelArch");
```

Comment:
Without this, there's tons of eval failures in `pkgs` in `release.nix`. This is from downstream portable packages depending on transitive `glibc`, since meta platform throwing is disabled.